prometheus_hardware_exporter/__main__.py

```diff
@@ -94,13 +136,20 @@ def main(port: int, level: str, enable_collectors: List[str], daemon: bool = Fal
 if __name__ == "__main__":  # pragma: no cover
     ns = parse_command_line()
     if ns.config:
-        config = Config.load_config(config_file=ns.config or DEFAULT_CONFIG)
+        exporter_config = Config.load_config(config_file=ns.config or DEFAULT_CONFIG)
```

Comment:
**nitpick:** 

`config` instead of `exporter_config` make more clean here, right?

Reply:
`config` is in `__main__` scope. mypy will complain about it, because `main()` function also use `config`

Reply:
It will result in a lint error in the `main()` function because we're re-defining `config` from the global scope in that function.